helix-term/src/ui/tree.rs

```diff
@@ -752,9 +752,9 @@ fn render_tree<T: TreeViewItem>(
     let indent = if level > 0 {
         let indicator = if tree.item().is_parent() {
             if tree.is_opened {
-                ""
+                "⏷"
             } else {
-                ""
+                "⏵"
             }
```

Comment:
Both chars, old and new, don't work with Ubuntu 22.04 default font.
Try  ▼ and  ▶

Reply:
[It's true](https://fonts.google.com/specimen/Ubuntu+Mono?preview.text=%EF%91%A0%E2%8F%B5%E2%96%BC%20and%20arrow_forward&preview.text_type=custom). Isn't that play_button emoji you typed?

Basically Ubuntu Mono only has [this much glyphs](https://fonts.google.com/specimen/Ubuntu+Mono/glyphs). I think it isn't good to base on this font for terminal application.

Reply:
It looks like a play_button on the browser, but on terminal it is just a triangle.
I guess my browser changed the glyph. I don't know why.

Try this on terminal: `echo "\u25bc and \u25b6"`

This is how it looks on helix: 
![image](https://github.com/helix-editor/helix/assets/1058103/02c6b79c-4840-4b3f-99e6-78de7cfe82be)

Ubuntu uses DejaVu Sans too, all these symbols work on terminal with the default font.
![image](https://github.com/helix-editor/helix/assets/1058103/cc5bcf2a-9f1a-48b7-82b1-79462207e3da)

You can try [here](https://design.ubuntu.com/font?ProyectoByte).
![image](https://github.com/helix-editor/helix/assets/1058103/8c1be3dd-1f31-4f6a-b0f1-86ac91e39924)

Try these too: 
"▾ and ▸"
`echo "\u25be and \u25b8"`

Reply:
Actually, I've been wondering if I should go with `V` and `>` for maximum compatibility, with a minute drop of aesthetic

Reply:
You could use standard Unicode symbols, these should be supported everywhere. Here you can find the arrows you are looking for: https://en.wikipedia.org/wiki/Miscellaneous_Symbols_and_Arrows